Guides/Python/HelloWorld.md

```diff
@@ -52,20 +78,20 @@ Delta compression using up to 4 threads.
 Compressing objects: 100% (10/10), done.
 Writing objects: 100% (16/16), 258.30 KiB, done.
 Total 16 (delta 2), reused 16 (delta 2)
-       
+
 -----> Receiving push
 -----> No runtime.txt provided; assuming python-2.7.3.
 -----> Preparing Python runtime (python-2.7.3)
 -----> Installing Distribute (0.6.36)
 -----> Installing Pip (1.3.1)
 -----> Installing dependencies using Pip (1.3.1)
-       Downloading/unpacking Flask==0.9 (from -r requirements.txt (line 1))
+       Downloading/unpacking Flask==0.10.1 (from -r requirements.txt (line 1))
```

Comment:
I haven't checked, but will the other version not most likely have changed also?